Source/WebCore/platform/graphics/egl/GLContext.cpp

```diff
@@ -356,21 +370,88 @@ RefPtr<GLDisplay> GLContext::display() const
     return m_display.get();
 }
 
+std::pair<unsigned, unsigned> GLContext::forcedEGLVersion()
+{
+    static std::once_flag onceFlag;
+    static unsigned majorVersion = 0;
+    static unsigned minorVersion = 0;
+
+    std::call_once(onceFlag, [] {
+        if (const char* envString = getenv("WEBKIT_FORCE_EGL_VERSION")) {
+            auto version = parseInteger<unsigned>(StringView::fromLatin1(envString)).value_or(0);
+            if (version >= 200 && version <= 320) {
+                majorVersion = version / 100;
+                minorVersion = (version % 100) / 10;
+                WTFLogAlways("WEBKIT_FORCE_EGL_VERSION=%d: Forcing OpenGL ES %d.%d", version, majorVersion, minorVersion);
+            } else
+                WTFLogAlways("WEBKIT_FORCE_EGL_VERSION=%d: Invalid version string, must be >= 200 <= 320", version);
+        }
+    });
+
+    return std::make_pair(majorVersion, minorVersion);
+}
+
 EGLContext GLContext::createContextForEGLVersion(EGLDisplay eglDisplay, EGLConfig config, EGLContext sharingContext)
 {
     WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN // GLib / Windows ports.
-    static EGLint contextAttributes[3];
+    static EGLint contextAttributes[7];
     WTF_ALLOW_UNSAFE_BUFFER_USAGE_END
```

Comment:
We can change this to `std::array` and then remove the `WTF_ALLOW_UNSAFE_BUFFER_USAGE_{BEGIN,END}` warning suppressions:

```suggestion
    static std::array<EGLint, 7> contextAttributes;
```